neqo-http3/src/lib.rs

```diff
@@ -470,6 +473,11 @@ trait RecvStream: Stream {
     fn webtransport(&self) -> Option<Rc<RefCell<WebTransportSession>>> {
         None
     }
+
+    /// This function is only implemented by `WebTransportRecvStream`.
+    fn stats(&mut self, _conn: &mut Connection) -> Res<RecvStreamStats> {
+        Err(Error::Unavailable)
```

Comment:
It seems to me like you could just make this:

```rust
Ok(conn.recv_stream_stats(self.stream_id)?)
```

It's not like the stats provided would be invalid for non-webtransport streams, only that you wouldn't be reporting adjusted values.

Reply:
Unfortunately, `self` doesn't have `stream_id` here.
I'll keep returning an error here.

Reply:
Cool.  No worries.